### PR TITLE
ER-827 Adds operation recruit apprentices

### PR DESCRIPTION
**Pull request by dev4valtech**

### CODE REVIEW

File: src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs [hunk 1]
```diff
@@ -32,14 +33,15 @@ public Task Get_WhenGettingGetAction_ThenShouldReturnGetView()
         {
             return RunAsync(f => f.Get(), (f, r) =>
             {
+                Debugger.Break();
```
Comment (elliotritchie) [inline src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs]: The breakpoint can be removed.

File: src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs [hunk 2]
```diff
@@ -32,14 +33,15 @@ public Task Get_WhenGettingGetAction_ThenShouldReturnGetView()
...
                 r.Should().NotBeNull().And.Match<ViewResult>(a => a.ViewName == "");
 
                 var model = r.As<ViewResult>().Model.Should().NotBeNull().And.BeOfType<GetAccountProviderLegalEntityViewModel>().Which;
 
                 model.AccountProvider.Should().BeSameAs(f.GetAccountProviderLegalEntityQueryResult.AccountProvider);
                 model.AccountLegalEntity.Should().BeEquivalentTo(f.GetAccountProviderLegalEntityQueryResult.AccountLegalEntity);
                 
-                model.Operations.Should().BeEquivalentTo(new List<OperationViewModel>
+                model.Operations.Take(1).Should().BeEquivalentTo(new List<OperationViewModel>
```
Comment (elliotritchie) [inline src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs]: I think it'd be worth asserting against the other items in the collection e.g.

```c#
model.Operations.Should().BeEquivalentTo(new List<OperationViewModel>
{
    new OperationViewModel
    {
        IsEnabled = true,
        Value = Operation.CreateCohort
    },
    new OperationViewModel
    {
        IsEnabled = false,
        Value = Operation.RecruitApprentices
    }
});
```

File: src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs (hunk 1 above)
Comment (dev4valtech) [reply src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs]: I wasn't able to debug, this was my attempt to get it working... sorry missed to take it out.

File: src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs (hunk 2 above)
Comment (dev4valtech) [reply src/SFA.DAS.ProviderRelationships.Web.UnitTests/Controllers/AccountProviderLegalEntitiesControllerTests.cs]: Done

File: src/SFA.DAS.ProviderRelationships.Types/Models/Operation.cs [hunk 3]
```diff
@@ -2,6 +2,7 @@
 {
     public enum Operation : short
     {
-        CreateCohort = 0
+        CreateCohort = 0,
+        RecruitApprentices = 1
```
Comment (sachpatel) [inline src/SFA.DAS.ProviderRelationships.Types/Models/Operation.cs]: Can this be changed to `CreateApprenticeshipVacancy`

Comment (dev4valtech) [reply src/SFA.DAS.ProviderRelationships.Types/Models/Operation.cs]: This is also the content that shows on the website and this is decided by Richard.

**Pull request closed without merging**